regression-tests.recursor-dnssec/test_Protobuf.py

```diff
@@ -129,7 +129,7 @@ def checkProtobufBase(self, msg, protocol, query, initiator, normalQueryResponse
         self.assertEquals(msg.id, query.id)
         self.assertTrue(msg.HasField('inBytes'))
         if normalQueryResponse:
-            # compare inBytes with length of query/response
+            # compare inBytes with length off query/response
```

Comment:
Wasn't `of` actually the right version?

Reply:
Indeed

Reply:
Yeah, only the other was correct, thanks